Casks/microsoft-git.rb

```diff
@@ -0,0 +1,20 @@
+cask 'microsoft-git' do
+  name 'Microsoft-fork of Git'
+  homepage 'https://github.com/microsoft/git'
+
+  version '2.27.0.vfs.1.0'
+  sha256 'f2d42987c21e110c2a5a9b4e8713c81eada38b0726f6b4fc85932ed32f71caf7'
+
+  url "https://github.com/microsoft/git/releases/download/v#{version}/git-#{version}-intel-universal-snow-leopard.pkg"
+
+  pkg "git-#{version}-intel-universal-snow-leopard.pkg", allow_untrusted: true
+
+  conflicts_with formula: 'git'
+
+  uninstall script: {
+            executable: '/usr/local/git/uninstall.sh',
+            args:       ['--yes'],
```

Comment:
Note that the uninstall script won't actually work because this `--yes` option is only being added in https://github.com/derrickstolee/git_osx_installer/pull/1, so need to wait for a new release.

Reply:
Will the argument just be ignored by the old installer? If so, then we can merge this and move onto the step that auto-updates the cask on a release in `microsoft/git`, right?

Reply:
The old uninstall script will ignore it (it does not reference `$1`).